ReText/tabactiontypes.py

```diff
@@ -0,0 +1,18 @@
+from enum import StrEnum
```

Comment:
StrEnum was introduced in Python 3.11, so if you want to use that, we would need to bump required Python version to 3.11. Please do that in a separate PR.

Or use simple Enum.